src/main/java/org/isf/menu/gui/UserBrowsing.java

```diff
@@ -149,6 +130,7 @@ public UserBrowsing() {
 		buttonPanel.add(buttonNew);
 
 		JButton buttonEdit = new JButton(MessageBundle.getMessage("angal.common.edit.btn"));
+		buttonEdit.setEnabled(false);
```

Comment:
I don't think we should disable this button, because one may need to restore the user, hence we need to update the UserEdit window with the new checkbox